src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
@hellobrian @rahulrkr08 If you want to opt-in/out the “traling slash removal” feature, you can add a class to breadcrumb instead of breadcrumb item. Again adding class to breadcrumb item for this kind of feature forces consumers to look at where the last breadcrumb items is to add a class.

Below is an example of what I'd suggest:

```css
.bx--breadcrumb-item__without-trailing-slash .bx--breadcrumb-item:last-child::after {
  content: '';
}
```